inventory/group_vars/_example.yml

```diff
@@ -10,3 +10,9 @@ language_packages:
   - language-pack-en-base
 l10n_repo: https://github.com/openfoodfoundation/l10n_au.git
 timezone:
+
+developer_email: <dev email>
```

Comment:
Please check new commit. I'd like to handle the privacy issue by introducing the use of Ansible Vault, if thats ok. We need to have various properties stored secretly and Vault is probably the simplest way to do that?